config/initializers/diaspora_federation.rb

```diff
@@ -106,7 +106,20 @@
       end
     end
 
-    on :save_entity_after_receive do
+    on :receive_entity do
+      # TODO
+    end
+
+    on :fetch_public_entity do |entity_type, guid|
+      entity = entity_type.constantize.find_by(guid: guid, public: true)
+      Diaspora::Federation.post(entity) if entity.is_a? Post
+    end
+
+    on :fetch_person_url_to do |diaspora_id, path|
+      Person.find_by(diaspora_handle: diaspora_id).send(:url_to, path)
```

Comment:
This is only for backporting and can be changed to
```
Pod.joins(:people).find_by(people: {diaspora_handle: diaspora_id}).url_to(path)
```
in `develop`-branch later.